questionnaires/ralpmh_covid_symptoms/ralpmh_covid_symptoms_armt.json

```diff
@@ -0,0 +1,648 @@
+[
+    {
+        "field_name": "fever",
+        "form_name": "ralpmh_covid_symptoms",
+        "section_header": "Do you have any COVID-19 Symptoms?",
+        "field_type": "matrix-radio",
+        "field_label": "fever",
```

Comment:
@yatharthranjan I think it would good to have the field labels either all capitalised or lowercase (since it is currently mixed)?

Reply:
Yes, good spot. it is because it was combined from 2 different questionnaires (WHO + CDC), 😅
I have made all of these uppercase. Can you run the script again? Thanks.